drizzle-orm/src/libsql/session.ts

```diff
@@ -1,6 +1,6 @@
-import { type Client, type InArgs, type InStatement, type ResultSet, type Transaction } from '@libsql/client';
+import type { Client, InArgs, InStatement, ResultSet, Transaction } from '@libsql/client';
+import { entityKind, is } from '~/entity.ts';
```

Comment:
I think this import shouldn't have changed, `is` isn't used anywhere in the file